Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20-alpine
+
+WORKDIR /usr/app
+
+COPY . .
+
+RUN yarn
+
+EXPOSE 3000
```

Comment:
it's good practice to specify `CMD`:
```
CMD [ "yarn", "start" ]
```
you still have an option to override it in docker-compose